### ISSUE TITLE
node version

### ISSUE THREAD

**Issue by Giancarlo1974** [phillipj/gitlab-search#40]
what is the minimum version of node for this project?

**Comment by phillipj** [comment]
Good question for which I don't know the answer of. Haven't been tested
across versions by me.

Honestly haven't used this tool for several years, so I can only suggest
you try it out and see if it explodes or not.

**Comment by Giancarlo1974** [comment]
Do you remember which version of Gitlab you developed this tool for?
I ask because our Gitlab, when interrogated by the tool explodes with error 500

Why don't you use this tool anymore?
Did you find another free solution?

**Comment by phillipj** [comment]
Nope, sorry I don't remember the Gitlab version either. Assuming you can
correlate it with the commit history of the project.

I haven't used it personally for a while because I changed employer and
thereby my use of Gitlab stopped unfortunately (as Gitlab is great).
